doc/api/n-api.md

```diff
@@ -4596,6 +4596,7 @@ idempotent.
 
 This API may only be called from the main thread.
 
+[ABI Stability]: ../guides/abi-stability.md
```

Comment:
It seems this should be an absolute link URL to work from the site.

https://nodejs.org/en/docs/guides/ is provided by the https://github.com/nodejs/nodejs.org/tree/master/locale/en/docs/guides , not by the https://github.com/nodejs/node/tree/master/doc/guides .